admin_app/src/app/dashboard/components/Overview.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
I am starting to keep components next to the endpoint/page as per [this page](https://alexkondov.com/tao-of-react/).

We'll refactor the others to be the same at some stage